application.py

```diff
@@ -41,6 +42,11 @@ def load_configuration():
             "user": "",
             "password": "",
             "limit": "35",
+            "components": "",
+            "priorities": "lowest,low,normal,high,highest",
+            "types": "bug,feature,task",
```

Comment:
Are you wanting these to be useful defaults for anyone using this rather than our use? The Trac defaults are "defect", "enhancement" and "task".